sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -19,6 +19,115 @@
      */
     public static final Configuration NONE = new NoopConfiguration();
 
+    // Default properties - these are what we read from the environment
+    /**
+     * URI of the proxy for HTTP connections.
+     */
+    public static final String PROPERTY_HTTP_PROXY = "HTTP_PROXY";
```

Comment:
What if the user has a proxy configured, but also requires authentication? Is there any username, password, or authentication type (digest vs basic vs none) variable name we should use?

Reply:
I don't believe we have anything defined in the spec for this.

Reply:
Hmm.. Feels like a gap in the spec, but part of another conversation. :)

Reply:
Yes, please do chase up on it separately so we can get it clarified, if necessary.